shared/utils/verify-references.py

```diff
@@ -205,6 +209,22 @@ def main():
                 print "No severity assigned to XCCDF Rule: " + rule.get("id")
                 exit_value = 1
 
+    if options.rules_without_cce or options.all_checks:
+        for rule in rules:
+		# find all CCEs in current rule
+		cces = rule.findall(".//{%s}ident" % xccdf_ns)
```

Comment:
This should be enhanced like follows:
```
cces = rule.findall(".//{%s}ident[@system=\"%s\"]" % (xccdf_ns, cce_ref_href))
```
to search just CCE ```ident``` elements. (The current version is searching all ```ident``` elements, thus e.g. also something like:
```
<ident system="http://iase.disa.mil/stigs/os/unix-linux/Pages/index.aspx">DISA FSO RHEL-06-000013</ident>
```
will match the condition, and ```cces``` won't be None. Thus no warning will be printed for that rule.